README.md

```diff
@@ -1,6 +1,6 @@
 # anki-slides-import
 
-Import pdf slides + text notes into [Anki](http://ankisrs.net/).
+This is a fork of [Utkarsh Upadhyay's](https://github.com/musically-ut) [anki-slides-import](https://github.com/musically-ut/anki-slides-import) code for importing PDF slides + text notes into [Anki](http://ankisrs.net/) that includes some additional functionality.  This modified version allows for the PDF slides to be displayed either in the Anki card question or answer, to be included alongside some additional text either before or after the slide, or to be left out completely.
```

Comment:
Usually these commits stay on the `master` branch while the other commits (_i.e._ the changes) are done on a separate `feat-*` or a `fix-*` branch, on which you create a PR. Could you please move these commits to a separate branch before I merge? 

Otherwise, you can create a separate branch based on `musically-ut:master`, `cherry-pick` the relevant commits and then create a separate PR without the changes to the URLs in the README.md.